src/app/todolist/domain/todo.ts

```diff
@@ -2,4 +2,10 @@ export class Todo {
   id: number;
   title: string;
   done: boolean;
+
+  constructor() {
+    this.id = 0;
+    this.title = '';
+    this.done = false;
+  }
```

Comment:
- あんまかわんないけど変数定義する時に初期値いれちゃえばいいかな？
- あとidは自動採番してるから`id?: number`でnull許可してもいいかもね

```suggestion
  id = 0;
  title = '';
  done = false;
```

Reply:
なるほど、修正しました！

Reply:
https://github.com/chisackeeey/angular-handson/pull/9#discussion_r436418065